sprint0/sprint0/Link/Sprites/LeftWalkingLinkSprite.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System.Collections.Generic;
+
+namespace sprint0
+{
+    class LeftWalkingLinkSprite : ISprite
+    {
+        public Texture2D Texture { get => texture; set => texture = value; }
+        public Vector2 Location { get => location; set => location = value; }
+
+        private Texture2D texture;
+        private Vector2 location;
+        private List<Rectangle> frames;
+        private int currentFrame;
+        private int repeatFrames;
+        private int totalFrames;
+
+        public LeftWalkingLinkSprite(Texture2D texture, Vector2 location)
+        {
+            this.texture = texture;
+            this.location = location;
+            currentFrame = 0;
+            repeatFrames = 8;
+            totalFrames = 2 * repeatFrames;
```

Comment:
I like this better than the multiplying and dividing every time I was doing before; we should refactor all of the sprites like this